routes/index.js

```diff
@@ -22,11 +22,33 @@ module.exports = function(passport){
   router.get('/register', function(req, res){
     res.render('register.njk');
   });
-
-  router.post('/register', passport.authenticate('register', {
-    successRedirect: '/confirmation',
-    failureRedirect: '/register'
-  }));
+  
+  /*post from registration form, validate and authenticate user input*/
+  router.post('/register', function(req,res,next) {
+	req.check('email','Not a valid email address').isEmail();
+	req.check('password','password must be atleast 5 characters').isLength({min:5});
+	req.check('reenterpassword','both passwords must match').equals(req.body.password);
+	var errors = req.validationErrors();
+
+	if(errors)
+	{
+		console.log(errors);			
```

Comment:
Remove debugging info, passwords potentially printed to node console running on server